### ISSUE TITLE
[AzureTools][Aspire] The scroll bar of the page is not responding when scrolling the mouse wheel down or up

### ISSUE THREAD

**Issue by v-mengwe** [dotnet/aspire#1707]
**Clean machine:** Win11 x64 23h2 ENU
**VS Version:** VS 17.10 Preview 1 from [34516.161.main]
**Aspire Version:** 8.0.0-preview.3.24066.3
Apply NuGet Feeds

**REPRO STEPS:**

1. Create a new 8.0 project with "Enlist in .NET Aspire orchestration" checked.
2. F5
3. In the dashboard, click "Metrics" -> Select a resource -> Drag the scroll bar of the page to the bottom.
4. Click the "Filters" icon in the Filters -> Expand a filter window and the page scroll bar is no longer at the bottom.
![image](https://github.com/dotnet/aspire/assets/62053665/d34fc71e-5aea-424b-90b4-a4b95f7d51c2)
5. The mouse focuses on the page -> Scroll mouse wheel down or up.
6. Check whether the scroll bar of the page is responding.

**Expect:**
The scroll bar of the page is responding.

**Actual:**
The scroll bar of the page is not responding.
https://github.com/dotnet/aspire-samples/assets/62053665/6f283670-44fa-4ccb-bf5f-dabfa8d3ba3f

**More Info:**
The mouse focuses on the opened filter window -> Scroll mouse wheel down -> The scroll bar of the page is responding.

**Comment by adamint** [comment]
dependent on #1755

**Comment by kvenkatrajan** [comment]
@v-mengwe can you retest and close - I am unable to repro this after the feature : https://github.com/dotnet/aspire/pull/1755

**Comment by davidfowl** [comment]
Assuming we can close this right?

**Comment by v-mengwe** [comment]
Verified on the rel.d17.10-34701.141 with .NET Aspire 8.0.0-preview.4.24129.7, this issue still reproduces.
![Actua](https://github.com/dotnet/aspire/assets/62053665/e7d62029-95aa-41a0-9e06-09a4f1d9f1f2)

**Comment by tlmii** [comment]
I can repro this still too. I think its because the FluentOverlay that's behind the popover is there to allow light dismiss (e.g. when you click anywhere else other than the popover it dismisses the popover). But that also means that when the mouse is over that and you try to use the scroll wheel, it'll get ignored (since the overlay div can't be scrolled). When the mouse is over the popover, it can (the popover is not a descendant of the overlay, it's a sibling with a higher z-index).

I'm not sure if its possible to get the overlay to let the mouse wheel actions through while still catching the clicks. @vnbaaij have you run into this before?
